nevergrad/functions/multiobj/__init__.py

```diff
@@ -0,0 +1,140 @@
+from typing import Callable, List
+from abc import ABC
+import nevergrad
+import numpy as np
+from nevergrad.parametrization import parameter as p
+
+
+class ConstrainedMultiObjective(ABC):
+    def __init__(
+        self,
+        name: str,
+        parametrization: p.Data,
+        objective_funcs: List[Callable[[np.ndarray], float]],
+        constraint_funcs: List[Callable[[np.ndarray], bool]],
+        constraints_as_layer: bool = False,
+    ) -> None:
+        super().__init__()
+        self.name: str = name
+        self.parametrization: p.Data = parametrization
```

Comment:
why not just `Array`? it shouldnt change anything anyway, but it's a bit weird
another option would have been to use Instrumentation(x=Scalar(lower, upper), y=Scalar(lower, upper)), which would let you use `x` and `y` directly in the definitions. It may improve readibility, but not sure.